web/pgadmin/tools/schema_diff/static/js/components/SchemaDiffComponent.jsx

```diff
@@ -82,6 +92,10 @@ export default function SchemaDiffComponent({params}) {
     });
   }
 
+  useEffect(()=>{
+    initializeSchemaDiff();
```

Comment:
I think we should show a spinner or something till initialise in progress.

Reply:
Sure will work on it.

Reply:
As per offline discussion, it is not required.